x-pack/plugin/sql/jdbc/src/main/java/org/elasticsearch/xpack/sql/jdbc/TypeConverter.java

```diff
@@ -342,6 +349,12 @@ private static Byte asByte(Object val, EsType columnType, String typeString) thr
             case INTEGER:
             case LONG:
                 return safeToByte(((Number) val).longValue());
+            case UNSIGNED_LONG:
```

Comment:
I've extracted that into a method.
(My initial reason not too is to avoid the extra check, given that the switch jumps already at the right point.)
`byteValueExact()` will raise an exception if truncation would occur, which is what we want for the method to be "safe".